packages/components/src/menu-group/index.js

```diff
@@ -19,6 +19,10 @@ export function MenuGroup( {
 	className = '',
 	instanceId,
 	label,
+	role = 'menu',
+	labelRole = ( role === 'group' ) ? { role: 'presentation' } : {},
```

Comment:
I'd remove this, as role=presentation is useful to "flatten" the accessibility tree but there's no need for it, as the `div` it applies to is already ignored.